chroma-manager/tests/utils/remote_firewall_control.py

```diff
@@ -187,19 +187,28 @@ def process_rules(self):
         result = self.remote_access_func(self.address, self.firewall_list_cmd)
 
         if result.rc != 0:
-            from chroma_common.lib.shell import Shell
-            raise RuntimeError('''process_rules(): remote shell command failed unexpectedly (%s), is firewall-cmd running? (%s) (%s)
-systemctl status firewalld:
-%s
-
-systemctl status polkit:
-%s
-
-journalctl -n 100:
-%s''' % (result.rc, result.stdout, result.stderr,
-         Shell.run(['systemctl', 'status', 'firewalld']).stdout,
-         Shell.run(['systemctl', 'status', 'polkit']).stdout,
-         Shell.run(['journalctl', '-n', '100']).stdout))
+            if result.rc == 254 and ' is not registered' in result.stderr:
+                from time import sleep
+                # restart polkit to attempt to address https://bugzilla.redhat.com/show_bug.cgi?id=1221483
+                self.remote_access_func(self.address, 'systemctl restart polkit')
```

Comment:
Hrm.  So the version of firewalld that fixed the problem in Fedora 22 was 0.3.14.2-2 and what we have in EL7 currently is 0.4.3.2-8.1.

So unfortunately this might be different than [BZ#1221483](https://bugzilla.redhat.com/show_bug.cgi?id=1221483).

Hopefully the fixing of the information gathering will give us enough information to open a new RH ticket about it.  But even if the polkit restart does work-around the issue, we should still open a RH ticket, so we wouldn't want to let that work-around suppress gathering the info.

If we want to allow the polkit restart to work-around the issue and let the test pass the information gathering should probably be [handled](https://github.com/intel-hpdd/intel-manager-for-lustre/blob/f9ac131a062be7f1b5e8138ee79a7222d87d3022/chroma-manager/tests/integration/core/remote_operations.py#L401-L426) similar to #29 otherwise it will get masked by passing tests.